spec.md

```diff
@@ -191,24 +167,23 @@ Returns an array of transaction records in the range `[start, start + limit)`.
 public query func getTransactions(start: Nat, limit: Nat) : async [TxRecord]
 ```
 
-##### getUserTransactionAmount
```

Comment:
I would suggest removing this: the only way to implement this effectively is to have a separate map, mapping Principal -> count. We cannot iterate through all the transaction in a performant way.